frontend/simple/components/Group/GroupRules.vue

```diff
@@ -111,20 +120,24 @@
 </style>
 <script>
 import { CircleSlider } from 'vue-circle-slider'
+import { toPercent } from '../../filters'
 
-const SUPERMAJORITY = 67
+const SUPERMAJORITY = 0.67
 const OKCOLOR = '#78c848'
 const WARNCOLOR = '#f68b39'
```

Comment:
I wonder, should these `OKCOLOR` / `WARNCOLOR`s be somehow taken from the SASS/SCSS variables so as to not have them in two separate places?

I wonder if @sandrina-p might know how to do that?

Reply:
Opened issue #373, this should be fixed as a separate PR, perhaps by @sandrina-p